src/components/eip3643/User.tsx

```diff
@@ -47,22 +66,24 @@ export default function User() {
           }}
           variant="outline"
         >
-          {tokens.map((item) => (
+          {sortTokensList().map((item) => (
```

Comment:
this would run on every render, we can optimise a lil bit with useMemo

Reply:
> this would run on every render, we can optimise a lil bit with useMemo

Just right!